src/store/module/task.module.js

```diff
@@ -0,0 +1,40 @@
+const task = {
+  nameSpaced: true,
+  state () {
+    return {
+      todos: []
+    }
```

Comment:
Add a new state for storing items which need to be edited. And you can create respective actions, mutations and getters for the same. So that you can call the respective getter in your form component for showing values inside the form.